src/ui/public/vislib/lib/types/point_series.js

```diff
@@ -199,7 +199,7 @@ export function VislibTypesPointSeries() {
         },
         labels: {
           filter: false,
-          axisFormatter: data.get('zAxisFormatter') || function () { return ''; }
+          axisFormatter:  function (val) { return val; }
```

Comment:
the labels for y axis are derived from series labels which are already formatted, so no need to call fieldformatter again